integers_floats_challenge/temperature.py

```diff
@@ -0,0 +1,20 @@
+# Convert a temp of 100 degress (F) to (C)
+# Equaiton Celsius = (Fahrenheit - 32) *5/9
+
+#print('Celsius_100')
+#print('Celsius_0')
+Fahrenheit = (34.2-32) * .5559
```

Comment:
instead of `.5559` you can write `(34.2-32)*9/5`, which is more exact

Reply:
still need to fix this will. I have everything to you by Sunday